transport/tchannel/header.go

```diff
@@ -101,27 +79,14 @@ func writeRequestHeaders(
 	appHeaders map[string]string,
 	getWriter func() (tchannel.ArgWriter, error),
 ) error {
-	ctxHeaders := baggage.FromContext(ctx)
+	ctxHeaders := transport.Headers{}
```

Comment:
`ctxHeaders` isn't used anymore except to check its length a line below. You
can drop it and the length check.